eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```diff
@@ -3,6 +3,8 @@ parameters:
   PackageName: "" # For Rush toolchain only
   PackagePath: "" # For NPM toolchain only
   PreIntegrationSteps: ""
+  ProvisionScopedResources: false
+  ResourceFileLocation: ""
```

Comment:
I'm wondering whether we should opt for a convention based approach here. Where we have a ```test-resources``` (or similar) folder under each ```sdk/[service name]``` folder which the script scans for JSON files. For each JSON file it changes it creates a resource group and deploys it. That way I can introduce a new JSON file along side the existing one for experimentation purposes before folding it into the standard file.